app/main/middlewares/log_page_view.py

```diff
@@ -13,6 +13,8 @@ def wrapper(*args, **kwargs):
             "user_id": user_id,
             "route": request.url,
             "method": request.method,
+            "caller_function": route_function.__name__,
```

Comment:
Is there a reason we can't move most of this info from this decorator and put it directly in a AudiRequestFormatter. I think we can access all this info there. Can then differentiate between audit and app logs by having an AuditRequesatFormatter and AppRequestFormatter.

That way whenever someone uses the audit_logger, all audit info will be included by default and users of it won't need to remember. Saves us needing this intermediary configuration in this log_page_view decorator. Unless there is a reason audit_logger wouldn't always want all this info?

Reply:
Looking at moving it to the audit log which is a good idea